packages/RichTextEditing-Core.package/DocumentReadStream.class/instance/reset.st

```diff
@@ -0,0 +1,5 @@
+initialize
+reset
+	
+	"Set position to -1 so that first call to next will target beginning of text"
```

Comment:
since collections in squeak usually start at index 1 we should to that too

Reply:
This is necessary to allow noticing text structure at the beginning of the text without having to handle edge cases